app/models/mixins/supports_feature_mixin.rb

```diff
@@ -52,11 +52,46 @@ module SupportsFeatureMixin
   #
   extend ActiveSupport::Concern
 
+  QUERYABLE_FEATURES = {
+      :events => 'Query for events',
+      :control => '',
+      :live_migrate => '',
+      :evacuate => '',
+      :smartstate_analysis => '',
+      :retire => '',
+      :associate_floating_ip => '',
+      :disassociate_floating_ip => '',
+      :delete => '',
+      :resize => '',
+      :migrate => '',
+      :launch_cockpit => '',
+      :provisioning => '',
+      :reconfigure => '',
+      :regions => '',
+      :discovery => '',
```

Comment:
yeah, definitely. I havent sent rubocop over the commit yet. This is just me adding entries until the tests start :smile: